salt/etcd-proxy/etcd.conf

```diff
@@ -1,2 +1,10 @@
 [Service]
-ExecStartPre=/bin/bash -c "if [[ -d /var/lib/etcd/proxy ]] && [[ -d /var/lib/etcd/member ]]; then rm -rf /var/lib/etcd/member; fi"
+# follow these instructions:
+# https://coreos.com/etcd/docs/latest/etcd-live-cluster-reconfiguration.html#replace-a-failed-etcd-member-on-coreos-container-linux
+
+# remove previously created proxy directory
+ExecStartPre=/usr/bin/rm -rf /var/lib/etcd/proxy
+
+# NOTE: use this option if you would like to re-add broken etcd member into cluster
```

Comment:
What option is this referring to? It's already being done, no? Why clean up `/var/lib/etcd/proxy` twice?